multegula.go

```diff
@@ -118,6 +118,22 @@ func getMessage(nodes []messagePasser.Node, localNodeName string) messagePasser.
 	return messagePasser.Message{Source: localNodeName, Destination: dest, Content: content, Kind: kind}
 }
 
+/* receive message from PyBridge and send to messagePasser */
+func sendToMessagePasser() {
+    for {
+        message := bridges.ReceiveFromPyBridge()
+        messagePasser.Send(message)
+    } 
+}
+
+/* receive message from MessagePasser and send to PyBridge */
+func receiveFromMessagePasser() {
+    for {
+        message := messagePasser.BlockReceive()
```

Comment:
I see, yeah let's wait on that because we'll be making some major changes to the multicasting process and receiving messages and the receiveQueue will be a receiveChannel again which would be blocking so you don't have to worry about that anymore.